core/modules/views/includes/view.inc

```diff
@@ -1732,11 +1738,10 @@ class view {
   /**
    * Set up query capturing.
    *
-   * db_query() stores the queries that it runs in global $queries,
-   * bit only if dev_query is set to true. In this case, we want
-   * to temporarily override that setting if it's not and we
-   * can do that without forcing a db rewrite by just manipulating
-   * $conf. This is kind of evil but it works.
+   * db_query() stores the queries that it runs in global $queries, but only if
+   * dev_query is set to true. In this case, we want to temporarily override
+   * that setting if it's not, and we can do that without forcing a db rewrite,
```

Comment:
'db' should be 'DB'?

Reply:
Yup, it should. ...either that, or replace with the full word "database".